src/components/Link/src/Link.stories.mdx

```diff
@@ -0,0 +1,194 @@
+import { ArgsTable, Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import { Paragraph } from "@gemeente-denhaag/typography";
+import { ArrowLeftIcon, ArrowRightIcon } from "@gemeente-denhaag/icons";
+
+import { Link } from ".";
+import pkg from "../package.json";
+import readme from "../README.md";
+
+<Meta
+  title="Components/Navigation/Link"
+  component={Link}
+  parameters={{ componentSubtitle: `${pkg.name} - ${pkg.version}` }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Link" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{}}
+    parameters={{
+      docs: {
+        source: {
+          code: `
+import { Link } from "@gemeente-denhaag/link";
+import { Paragraph } from "@gemeente-denhaag/typography";
+const ReactParagraph = (
+  <Paragraph>
+    It is possible to put <Link href="#">a link</Link> in a Paragraph element.
+  </Paragraph>
+);
+// This 👆 transpiles into this 👇 in the DOM
+<p class="utrecht-paragraph utrecht-paragraph--distanced">
+  It is possible to put <a href="#" class="denhaag-link" tabindex="0">a link</a> in a Paragraph element.
+</p>
```

Comment:
We can remove this and use an HTML only story similar to what @mjcarsjens implemented on the Alert design PR